howtos/module/_posts/2016-06-30-Ten-ways-improve-modules.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+title:  "Ten ways to improve your PrestaShop modules!"
+subtitle: "Stop the bugs, get back to innovating!"
+date: 2016-07-01 11:07:11
+author: [ mdekker ]
+icon: icon-puzzle
+tags: [ howto, tutorial, modules ]
+published: false
+---
+
+PrestaShop has numerous features in the API, both hidden in its code as well as in the developer documentation, that allow you to create your own module in no time!
+Unfortunately I have noticed that many developers do not use these features to their advantage and I think it's time to change all that :)
+Hopefully, these tips will increase your development speed and save you lots and lots of time. And if not satisfied, you'll get your money back! (Oh, wait...)  
+
+## The time savers
```

Comment:
To me, having a "## The time savers" section means another section is coming up. If it's coming in another article, then I'd put it directly in the article's title.